### PR TITLE
GitHub Releases Automation

### LINKED ISSUE #70: Automate sending updates from the GitHub release page to the #updates channel

### PR DESCRIPTION
**Pull request by Ferotiq**
Closes #70

### CODE REVIEW

File: src/events/ready.ts [hunk 1]
```diff
@@ -22,50 +26,40 @@ export default {
...
     if (guild === undefined) {
+      console.log("Could not get guild");
+
       return;
     }
 
-    client.user.setActivity(`over ${getMemberCount(guild)} monkeys`, {
-      type: "WATCHING"
-    });
-
```
Comment (Ferotiq) [inline src/events/ready.ts:31]: I extracted all of this into separate functions, really should have done that in a separate PR but oh well

**Review by Miodec — APPROVED** [review]
Thanks
